components/shared/molecules/Form/SignInForm.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Form, Formik, FormikProps } from 'formik';
+import * as Yup from 'yup';
+
+import useSignIn from 'lib/apollo/hooks/actions/useSignIn';
+
+import Button from 'components/shared/atoms/Button';
+import FormField from 'components/shared/atoms/FormField';
+import Loader from 'components/shared/atoms/Loader';
+import { FormFieldType } from './forms.types';
+
+const FormContentWrapper = styled.div`
```

Comment:
Could you please move styles to `styled.ts` as is customary in the whole project